src/examples/turtlebot4/README.md

```diff
@@ -0,0 +1,207 @@
+
+      ```bash
+      cd rai
+      # for Ubuntu 22.04 jammy and ros2 humlbe
+      unzip -d src/examples/turtlebot4/simulation Turtlebot4_jammyhumble_0.0.1.zip
+
+      # for Ubuntu 24.04 noble and ros2 jazzy
+      unzip -d src/examples/turtlebot4/simulation Turtlebot4_noblejazzy_0.0.1.zip
+      ```
+
+3. Setup your LLM vendor: [docs/vendors.md](../../../docs/vendors.md). OpenAI or AWS Bedrock are recommended for now.
+
+4. Configure `rai_whoami_node` (based on ["Your robot identity in RAI"](https://github.com/RobotecAI/rai/blob/development/docs/create_robots_whoami.md) tutorial):
+
+   1. Create `whoami` package for turtlebot4 in `src/examples/turtlebot4`
+
+      ```bash
+      . ./setup_shell.sh
+      poetry run create_rai_ws --name turtlebot4 --destination-directory src/examples
+      ```
+
+   2. Download official turtlebot4 [data sheet](https://bit.ly/3KCp3Du) into
+      `src/examples/turtlebot4_whoami/description/documentation`
+   3. Download [image](https://s3.amazonaws.com/assets.clearpathrobotics.com/wp-content/uploads/2022/03/16113604/Turtlebot-4-20220207.44.png) of turtlebot4 into `src/examples/turtlebot4_whoami/description/images`
+   4. Create robot's identity. Run the `parse_whoami_package`. This will process the documentation, building
+      it into a vector database, which is used by RAI agent to reason about its identity.
+
+      > **NOTE**: generated files can be downloaded from [here](https://robotecai-my.sharepoint.com/:u:/g/personal/bartlomiej_boczek_robotec_ai/EbPZSEdXYaRGoeecu6oJg6QBsI4ZOe_mrU3uOtOflnIjQg?e=HX8ZHB)
+      > unzip them to `src/examples/turtlebot4_whoami/description/generated` with a command:
+      > `unzip -d src/examples/turtlebot4_whoami/description turtlebot4_whoami_generated.zip`
+
+      > **NOTE**: Vector database is created using the OpenAI API. Parsing bigger documents
+      > might lead to costs. Embedding model can be configured in
+      > [config.toml](https://github.com/RobotecAI/rai/blob/development/config.toml#L13)
+
+      ```bash
+      poetry run parse_whoami_package src/examples/turtlebot4_whoami
+      # you will be asked to press `y` to continue
+      ```
+
+   5. Build the workspace which now includes the new package
+
+      ```bash
+      deactivate # if poetry env is activated
+      colcon build --symlink-install
+      ```
+
+   6. Ensure `turtlebot4_whoami` package has been built:
+
+      ```bash
+      . ./setup_shell.sh
+      ros2 pkg list | grep turtlebot4
+      ```
+
+   Congratulations! Your `rai_whoami_node` is configured. In the following steps
+   your RAI agent will assume a turtlebot4 "personality".
+
+5. Run rai agent:
+
+   ```bash
+   ros2 launch ./src/examples/turtlebot4/turtlebot.launch.xml \
+       game_launcher:=./src/examples/turtlebot4/simulation/TurtleBot4DemoGamePackage/TurtleBot4Demo.GameLauncher
+   ```
+
+   You should be able to see a simulation scene.
+
+   > **Tip**: Press 1, 2 or 3 on the keyboard when simulation window to change the
```

Comment:
These shortcuts does not work for me - is this still available? WSAD camera control works fine.

Reply:
It might be caused by a bug in 24.09. Which binary did you use?